<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>432</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>PIP Image Processor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/images/images/logo.png</normaloff>:/images/images/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>704</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuImage">
    <property name="title">
     <string>Image</string>
    </property>
    <addaction name="actionAlgebra"/>
    <addaction name="actionBrightness"/>
    <addaction name="actionContrast"/>
    <addaction name="actionEqualization"/>
    <addaction name="actionFilter"/>
    <addaction name="actionGray_Scale"/>
    <addaction name="actionHistogram"/>
    <addaction name="actionRotate"/>
    <addaction name="actionScale"/>
   </widget>
   <widget class="QMenu" name="menuMorphology">
    <property name="title">
     <string>Morphology</string>
    </property>
    <widget class="QMenu" name="menuEdge_Detection">
     <property name="title">
      <string>Edge Detection</string>
     </property>
     <addaction name="actionStandard"/>
     <addaction name="actionInternal"/>
     <addaction name="actionExternal"/>
    </widget>
    <addaction name="actionMorpOper"/>
    <addaction name="actionDistance"/>
    <addaction name="actionSkeleton"/>
    <addaction name="actionReconstruct"/>
    <addaction name="menuEdge_Detection"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuImage"/>
   <addaction name="menuMorphology"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionGray_Scale"/>
   <addaction name="actionHistogram"/>
   <addaction name="actionEqualization"/>
   <addaction name="separator"/>
   <addaction name="actionAlgebra"/>
   <addaction name="actionScale"/>
   <addaction name="actionRotate"/>
   <addaction name="separator"/>
   <addaction name="actionFilter"/>
   <addaction name="actionContrast"/>
   <addaction name="actionBrightness"/>
   <addaction name="separator"/>
   <addaction name="actionMorpOper"/>
   <addaction name="actionDistance"/>
   <addaction name="actionSkeleton"/>
   <addaction name="actionReconstruct"/>
   <addaction name="actionStandard"/>
   <addaction name="actionInternal"/>
   <addaction name="actionExternal"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/images/open.png</normaloff>:/images/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionHistogram">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/images/histogram.png</normaloff>:/images/images/histogram.png</iconset>
   </property>
   <property name="text">
    <string>Histogram</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionFilter">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/images/filter.png</normaloff>:/images/images/filter.png</iconset>
   </property>
   <property name="text">
    <string>Filter</string>
   </property>
  </action>
  <action name="actionBrightness">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/images/bright.png</normaloff>:/images/images/bright.png</iconset>
   </property>
   <property name="text">
    <string>Brightness</string>
   </property>
  </action>
  <action name="actionContrast">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/images/contrast.png</normaloff>:/images/images/contrast.png</iconset>
   </property>
   <property name="text">
    <string>Contrast</string>
   </property>
  </action>
  <action name="actionScale">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/images/scale.png</normaloff>:/images/images/scale.png</iconset>
   </property>
   <property name="text">
    <string>Scale</string>
   </property>
  </action>
  <action name="actionAlgebra">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/images/algebra.png</normaloff>:/images/images/algebra.png</iconset>
   </property>
   <property name="text">
    <string>Algebra</string>
   </property>
  </action>
  <action name="actionEqualization">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/images/equal.png</normaloff>:/images/images/equal.png</iconset>
   </property>
   <property name="text">
    <string>Equalization</string>
   </property>
  </action>
  <action name="actionGray_Scale">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/images/gray.png</normaloff>:/images/images/gray.png</iconset>
   </property>
   <property name="text">
    <string>Gray Scale</string>
   </property>
  </action>
  <action name="actionRotate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/images/rotate.png</normaloff>:/images/images/rotate.png</iconset>
   </property>
   <property name="text">
    <string>Rotate</string>
   </property>
  </action>
  <action name="actionMorpOper">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/images/morphology.png</normaloff>:/images/images/morphology.png</iconset>
   </property>
   <property name="text">
    <string>Morphology Operation</string>
   </property>
  </action>
  <action name="actionSkeleton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/images/skeleton.png</normaloff>:/images/images/skeleton.png</iconset>
   </property>
   <property name="text">
    <string>Skeleton</string>
   </property>
  </action>
  <action name="actionDistance">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/images/distance.png</normaloff>:/images/images/distance.png</iconset>
   </property>
   <property name="text">
    <string>Distance</string>
   </property>
  </action>
  <action name="actionReconstruct">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/images/reconstruction.png</normaloff>:/images/images/reconstruction.png</iconset>
   </property>
   <property name="text">
    <string>Skeleton Reconstruction</string>
   </property>
  </action>
  <action name="actionStandard">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/images/standard.png</normaloff>:/images/images/standard.png</iconset>
   </property>
   <property name="text">
    <string>Standard</string>
   </property>
  </action>
  <action name="actionInternal">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/images/internal.png</normaloff>:/images/images/internal.png</iconset>
   </property>
   <property name="text">
    <string>Internal</string>
   </property>
  </action>
  <action name="actionExternal">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/images/external.png</normaloff>:/images/images/external.png</iconset>
   </property>
   <property name="text">
    <string>External</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
